app/components/Widget/Voting/Voting.js

```diff
@@ -0,0 +1,238 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { BigNumber } from 'bignumber.js'
+import {
+  Card,
+  GU,
+  IconCheck,
+  IconClose,
+  IdentityBadge,
+  ProgressBar,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { useNetwork } from '@aragon/api-react'
+import {
+  VOTING_STATUS,
+  VOTING_STATUS_ONGOING,
```

Comment:
Maybe we should use `Object.keys(VOTING_STATUS)` instead of being so redundant with the status definitions?